### PR TITLE
Static content style updates

### PR DESCRIPTION
**Pull request by barryclark**

### CODE REVIEW

File: lib/themes/dosomething/paraneue_dosomething/scss/feature/_content.scss [hunk 1]
```diff
@@ -3,49 +3,133 @@
...
 // ----------------
 
-// .wrapper {
-//   background: $purple;
-// }
+.node-type-static-content {
+  background: $purple;
 
-.static_content--wrapper {
+  // @TODO: White Navigation -- Ugh -- At least mixin this
```
Comment (DFurnes) [inline lib/themes/dosomething/paraneue_dosomething/scss/feature/_content.scss:9]: Totes.

File: lib/themes/dosomething/paraneue_dosomething/scss/feature/_content.scss [hunk 2]
```diff
@@ -3,49 +3,133 @@
...
   background: #fff;
 
   h2, h3, h4 {
     color: $purple;
   }
 
-  .header--wrapper, .intro--wrapper, .cta--wrapper, .additional-text--wrapper, .gallery--wrapper {
+  // Set gutters on mobile
+  .header-wrapper, .intro-wrapper, .cta-wrapper, .additional-text-wrapper, .gallery-wrapper {
+    @include outer-container-margin();
```
Comment (mmwtsn) [inline lib/themes/dosomething/paraneue_dosomething/scss/feature/_content.scss:34]: Won't this prevent full-width images on mobile?

**Comment by mmwtsn** [comment]
Looks good.

File: lib/themes/dosomething/paraneue_dosomething/scss/feature/_content.scss (hunk 2 above)
Comment (barryclark) [inline lib/themes/dosomething/paraneue_dosomething/scss/feature/_content.scss:34]: @mmwtsn Yea, haven't finished image stuff just yet.

**Pull request merged**